### PR TITLE
Lesson Assignments

### PR DESCRIPTION
**Pull request by ojspoko**
lesson 2 assignment 1 pull request

### CODE REVIEW

**Comment by ojspoko** [comment]
countdown.js + calculator.js

File: Lesson02/assignment-1.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+console.log(x * 2);
+
+const y = 42;
+
+console.log(y > 8);
+
+let i = 5;
+let o = '5';
+
+console.log((i-3) === o);
```
Comment (jimmynono) [inline Lesson02/assignment-1.js]: You'll want to put a space before and after the minus sign (like what you do in the console logs).

File: Lesson02/countdown.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+for(let x = 10; x > -1; x--) {
```
Comment (jimmynono) [inline Lesson02/countdown.js]: You'll want to add a space after the for keyword.

File: exercise-one.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+let newEnglandCities = ["boston", "portland", "portsmouth"];
```
Comment (ClovisI) [inline exercise-one.js]: Need to have a total of five strings:
"Creates an array, with at least 5 strings"

Comment (ClovisI) [reply exercise-one.js]: You should also place the following after your array's declaration:
console.log(newEnglandCities);

**Pull request closed without merging**